modules/standard/Random.chpl

```diff
@@ -775,9 +788,10 @@ module Random {
 
       /*
         Advances/rewinds the stream to the `n`-th value in the sequence.
-        The first value is with n=1.
+        The first corresponds to n=1. It is an error to call this
+        routine with n <= 0.
 
-        :arg n: The position in the stream to skip to.  Must be non-negative.
+        :arg n: The position in the stream to skip to.  Must be > 0.
```

Comment:
I was thinking that NPB was different in this regard, but it turns out its documentation is just wrong too.  Could you fix its documentation as well?  (sorry, I very well was the person who introduced this mistake).